css-sizing-4/Overview.bs

```diff
@@ -569,6 +569,15 @@ Overriding Contained Intrinsic Sizes: the 'contain-intrinsic-*' properties</h3>
 			Otherwise,
 			its [=explicit intrinsic inner size=]
 			is the specified <<length>>.
+
+		: <dfn>auto none</dfn>
+		:: For a given axis, if the element has a [=last remembered size=],
+			and is currently [=skipping its contents=],
+			its [=explicit intrinsic inner size=] for the axis
+			is the [=last remembered size=].
```

Comment:
To avoid repeating this, it may be better to define the property with

```
Value: auto? [none | <<length>>]
```

and then just define the `auto [none | <<length>>]` cases together.